docs/sources/setup/install/helm/reference.md

```diff
@@ -7523,81 +7523,6 @@ false
   ]
 }
 </pre>
-</td>
```

Comment:
is this expected that the sections is getting removed?

Reply:
to be honest i didn't read through this file change at all, i just ran the `make helm-docs` command.  I can check into this more tomorrow.

Reply:
hmm i see.  so it's probably the difference between `# comment` and `# -- comment` that triggers this.  Do we have a preference which way to generate docs?  Personally I would lean away from documentation every type=map object is documented.  It leads to huge sections in the file like this
<img width="2688" height="1110" alt="image" src="https://github.com/user-attachments/assets/abaa0347-1307-4fba-8a72-f8303f3283ae" />

Reply:
yeah, it's true.  first time ever really getting into helm-docs.  by default it does not document lists and maps that have elements.  because i removed the `# --` it removed the section.
[MAKE](https://github.com/norwoodj/helm-docs?tab=readme-ov-file#valuesyaml-metadata)

Reply:
reverted